Src/Tests/Temporal.Sdk.Common.Tests/Serialization/TestCompositePayloadConverter.cs

```diff
@@ -0,0 +1,66 @@
+using Temporal.Api.Common.V1;
+using Temporal.Common;
+using Temporal.Common.Payloads;
+using Temporal.Serialization;
+using Xunit;
+
+namespace Temporal.Sdk.Common.Tests.Serialization
+{
+    public class TestCompositePayloadConverter
+    {
+        [Fact]
+        public void Test_CompositePayloadConverter_Void_Roundtrip()
+        {
+            CompositePayloadConverter converter = new CompositePayloadConverter();
+            Payloads p = new Payloads();
+            Assert.True(converter.TrySerialize(new IPayload.Void(), p));
+            Assert.Empty(p.Payloads_);
+            Assert.True(converter.TryDeserialize(p, out IPayload.Void item));
+            Assert.NotNull(item);
+        }
+
+        [Fact]
+        public void Test_CompositePayloadConverter_Null_Roundtrip()
+        {
+            CompositePayloadConverter converter = new CompositePayloadConverter();
+            Payloads p = new Payloads();
+            Assert.True(converter.TrySerialize<string>(null, p));
+            Assert.Single(p.Payloads_);
+            Assert.True(converter.TryDeserialize(p, out string item));
+            Assert.Null(item);
+        }
+
+        [Fact]
+        public void Test_CompositePayloadConverter_Unnamed_Roundtrip()
+        {
+            UnnamedContainerPayloadConverter unnamed = new UnnamedContainerPayloadConverter();
+            unnamed.InitDelegates(new[] { new JsonPayloadConverter() });
+            CompositePayloadConverter instance = new CompositePayloadConverter(new IPayloadConverter[]
+            {
```

Comment:
This test was written against the old converter. That was before we had the newtonsoft converter, so it always failed because it went to the catchall converter. That is no longer an issue, so I updated it.

Reply:
go it, thanks